src/md/resources.md

```diff
@@ -0,0 +1,71 @@
+
+## Presentations
+
+- Ilya Genelin - "Next gen decision making < 2ms" - 02/26/2016 - Apache Apex meetup
```

Comment:
All link URL's should be combined with content to make the list shorter and easier to read.